packages/core/src/metadata/di.ts

```diff
@@ -268,6 +281,10 @@ export interface ViewChildrenDecorator {
    *
    * * **selector** - The directive type or the name used for querying.
    * * **read** - Used to read a different token from the queried elements.
+   * * **emitDistinctChangesOnly** - True to only fire changes if there are underlying changes to
+   *   the query. This will become `true` be default in v12. `QueryList.changes` would fire even if
+   * no changes to the query list were detected. This is not ideal, as changes should only fire if
+   *   the `QueryList` actually materially changed.
```

Comment:
Consider something simpler like (might require reformatting):

```suggestion
   * * **emitDistinctChangesOnly** - The ` QueryList#changes` observable will emit new values only if the QueryList result has changed. 
   *     The default value will change from `false` to `true` in v12.
   *     When `false` the `changes` observable might emit even if the QueryList has not changed.
```